packages/forklift-console-plugin/package.json

```diff
@@ -45,7 +45,7 @@
     "react-router-dom": "5.3.x"
   },
   "devDependencies": {
-    "@kubev2v/types": "0.0.12",
+    "@kubev2v/types": "0.0.15",
```

Comment:
did you rebase ?

https://github.com/kubev2v/forklift-console-plugin/blob/main/packages/forklift-console-plugin/package.json#L48